crates/relbox/src/tx/transaction.rs

```diff
@@ -20,7 +20,7 @@ use std::thread::yield_now;
 use thiserror::Error;
 
 use moor_values::util::{BitArray, Bitset64};
-use moor_values::util::{PhantomUnsend, PhantomUnsync, SliceRef};
```

Comment:
Am completely lost here :D Noting that I didn't know / expect transactions to be non-`Send`, and I also don't understand why we now get to `Send` them. (same for the working set)

Reply:
So yeah, this I think was a bug in kanal. It was letting me `Send` even tho the thing in question was `!Send`. 

I mistakenly had these all `!Send` when they only needed to be `!Sync`.  The gist here is:

All state in a transaction belongs to a single thread (at a time). 
When it's committed, it gets sent (over channel) into the commit queue, who then takes ownership and processes through it.

Kanal was letting me do the latter even tho I had marked the item as non-Sendable. Crossbeam didn't like that, obviously.

Reply:
Idea: If you wanted to represent "don't Send until ready to commit", we could use an OpenTransaction wrapper that only gets unwrapped just before submitting to the commit thread. I don't think (low confidence) that this is worth the effort.

Reply:
So the way I think this should be is that it's just the `WorkingSet` that's `Send`. The Transaction should be `!Send`. When the transaction commits, the working set is taken from the transaction and sent over the channel. 

And it looks like I was just a bit overzealous here with removing Send from Transaction, and in fact that's entirely possible. So adding it back.